tests/bluetooth/bsim/audio/src/tbs_test.c

```diff
@@ -69,6 +110,257 @@ static struct bt_conn_cb conn_callbacks = {
 	.disconnected = disconnected,
 };
 
+static int test_provider_name(void)
+{
+	int err;
+
+	printk("%s\n", __func__);
+	err = bt_tbs_set_bearer_provider_name(0, "BabblesimTBS");
+	if (err != BT_TBS_RESULT_CODE_SUCCESS) {
+		FAIL("Could not set bearer provider name: %d\n", err);
+		return err;
+	}
+
+	printk("Set bearer provider name test success\n");
+
+	return err;
+}
+
+static int test_set_signal_strength(void)
+{
+	int err;
+
+	printk("%s\n", __func__);
+	err = bt_tbs_set_signal_strength(0, 6);
+	if (err != BT_TBS_RESULT_CODE_SUCCESS) {
+		FAIL("Could not set bearer provider name: %d\n", err);
+		return err;
+	}
+
+	printk("Set signal strength test success\n");
+
+	return err;
+}
+
+static int test_set_bearer_technology(void)
+{
+	int err;
+
+	printk("%s\n", __func__);
+	err = bt_tbs_set_bearer_technology(0, 6);
+	if (err != BT_TBS_RESULT_CODE_SUCCESS) {
+		FAIL("Could not set bearer technology: %d\n", err);
+		return err;
+	}
+
+	printk("Set bearer technology test success\n");
+
+	return err;
+}
+
+static int test_set_status_flags(void)
+{
+	int err;
+
+	printk("%s\n", __func__);
+	err = bt_tbs_set_status_flags(0, 3);
```

Comment:
Hmm, I guess we are missing definitions for the status flags, right?

Reply:
Will you add them as part of this PR, or as a Github issue?

Reply:
I'll prefer to add those in a new PR.